plugin/completion/lib_complete.py

```diff
@@ -311,7 +334,14 @@ def _parse_completions(complete_results):
             list: updated completions
         """
         completions = []
+        excluded = [Completer.cindex.CursorKind.DESTRUCTOR, Completer.cindex.CursorKind.CLASS_DECL]
+        if trigger != "::":
+            excluded.append(Completer.cindex.CursorKind.ENUM_CONSTANT_DECL)
```

Comment:
What and why exactly is that excluding? Why would it be wrong to list enum after double colon?
This case seem valid to me:

```
class A {
 public:
  enum EnumType {
    VALUE1,
    VALUE2,
  }
};

int main() {
  A::EnumType value = A::  // expecting to get VALUE1 and VALUE2 here
}
```

or same with `enum class`.